validation/td-validation.ttl

```diff
@@ -1728,6 +1714,20 @@
         sh:maxCount 1 ;
         sh:order 2 ;
     ] ;
+    sh:property [
+        sh:path jsonschema:contentEncoding ;
+        skos:definition """Specifies the encoding used to store the contents, as specified in RFC 2054. The values that are accepted: \"7bit\", \"8bit\", \"binary\", \"quoted-printable\" and \"base64\"."""^^rdf:HTML ;
+        sh:datatype xsd:string ;
+        sh:maxCount 1 ;
+        sh:order 14 ;
+    ] ;
+    sh:property [
+        sh:path jsonschema:contentMediaType ;
+        skos:definition """Specifies the MIME type (e.g., image/png, audio/mpeg) of the contents of a string value, as described in RFC 2046."""^^rdf:HTML ;
+        sh:datatype xsd:string ;
+        sh:maxCount 1 ;
+        sh:order 15 ;
+    ] ;
```

Comment:
I think those 2 properties should come after jsonschema:pattern.
see https://github.com/w3c/wot-thing-description/blob/4e25ce3fe80dde6eb613f37dd7f8aa39fea24057/validation/td-validation.ttl#L1731

I would also change the sh:order to 4 and 5 so that it matches the sequence...

Reply:
Right! fixed 👍🏻